assets/app/view/game_page.rb

```diff
@@ -265,6 +274,9 @@ def menu
           role: 'navigation',
           'aria-label': 'Game',
         },
+        hooks: {
+          postpatch: scroll_to_game_menu,
+        },
```

Comment:
There’s a problem with this hook. As far as I understood it hooks fire if something happens to "their" DOM node. So the postpatch hook here should fire after menu was patched. Yet, it fires e.g. on any use of a map controls button. Why?

Reply:
i think it happens anything is rerendered